### DISCUSSION TITLE
Option to disable inverted Y-axis when aiming

### DISCUSSION THREAD

**Discussion by jordanbtucker** [HarbourMasters/2ship2harkinian#507] [Ideas]
When aiming, for example when shooting a bubble as Deku Link, the Y-axis is inverted. There should be an option to disable this like in Ship of Harkinian.

Note that there is an option to disable inverted Y-axis for the free look camera, however this does not affect the Y-axis when aiming.

**Comment by Parandreu** [comment]
> Note that there is an option to disable inverted Y-axis for the free look camera, however this does not affect the Y-axis when aiming.

Where is this option?

**Reply by smashbro596** [reply]
in enhancements, under free look.
doesnt work with aiming your weapons.
